documentation/concepts/index.markdown

```diff
@@ -10,52 +10,36 @@ wordpress_id: 194
 
 # Concepts
 
-The following is an overview of how MoveIt! works. For more concrete documentation and details see the [tutorials](http://docs.ros.org/indigo/api/moveit_tutorials/html/)
+The following is an overview of how MoveIt! works. For more concrete documentation and details see the [tutorials](http://docs.ros.org/kinetic/api/moveit_tutorials/html/)
 
 ## System Architecture
 
 <img src='/wordpress/wp-content/uploads/2013/12/Overview.0012.jpg' style="width: 100%;"/>
 
 ### The move_group node
 
-
-The figure above shows the high-level system architecture for the primary node provided by MoveIt! called _move_group_. This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use.
-
+The figure above shows the high-level system architecture for the primary node provided by MoveIt! called [move_group](http://docs.ros.org/jade/api/moveit_ros_move_group/html/move__group_8cpp.html). This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use.
```

Comment:
i considered that, but decided that improving documentation is already such a hurdle for most people that waiting would certainly result in me just not making these improvements.